src/main/java/no/unit/nva/cristin/projects/CristinHandler.java

```diff
@@ -14,9 +17,19 @@
 
     protected static final String DEFAULT_LANGUAGE_CODE = "nb";
     private static final Set<String> VALID_LANGUAGE_CODES = Set.of("en", "nb", "nn");
+    private static final Set<String> VALID_CONTENT_TYPES = Set.of("application/json", "application/ld+json");
```

Comment:
can we add these types in commons?

Reply:
One of them already is but the other, ld json is missing. Should I commit to commons or somebody else? I have never worked with commons before

Reply:
Add it to the commons